core/src/main/scala/org/apache/spark/sql/TiExtensions.scala

```diff
@@ -65,8 +65,8 @@ object TiExtensions {
       .find(pair => TiCatalog.className.equals(pair._2)) match {
       case None =>
         logger.error(
-          "TiSpark is not work in catalog plugin mode! Please add catalog config in spark conf")
-        throw new TiInternalException("TiSpark is not work in catalog plugin mode")
+          "TiSpark must work in catalog plugin mode! Please add catalog config in spark conf")
```

Comment:
Perhaps `catalog plugin mode` is a deprecated description.
TiSpark must work with TiCatalog. Please add TiCatalog in spark conf.